setup.py

```diff
@@ -182,7 +182,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'atlasclient>=0.1.2',
 ]
 aws = [
-    'boto3~=1.10',
+    'boto3>=1.12.0,<2.0.0',
```

Comment:
Why `>=1.12.0` not `>=1.10.0` ? Is there an issue with `~=1.10` ?

Reply:
To use the newer version.

**Idea being**: Use newer version if it doesn't break anything